modules/vaos/app/models/vaos/message_form.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+require 'active_model'
+require 'common/models/form'
+
+module VAOS
+  class MessageForm < Common::Form
+    attribute :message_text, String # only this attribute gets set by user, everything else is overridden
```

Comment:
Who/what overrides this?

Reply:
So the form object receives input from our FE, this field is validated. But there are other attributes that need to get passed to VAMF for the request to validate on their side. Those other attributes are overridden such that the FE cannot mass-assign them but do get included in the form object as part of the outbound request payload.